rten-cli/src/main.rs

```diff
@@ -293,7 +293,8 @@ fn print_input_output_list(model: &Model, node_ids: &[NodeId]) {
 /// generated inputs.
 ///
 /// ```
-/// tools/convert-onnx.py model.onnx output.rten
+/// pip install -e rten-convert
```

Comment:
In the instructions in examples and other tools I think it will make sense to assume the user has followed the basic installation steps in the main README, so we don't need to duplicate the instructions for installing `rten-convert` in each example. This was already assumed to be the case for the dependencies of `tools/convert-onnx.py`.

Reply:
I removed unnecessary installation references